evm/p2p/peer.py

```diff
@@ -449,6 +452,23 @@ def les_proto(self):
         return self._les_proto
 
 
+class ETHPeer(BasePeer):
+    _eth_proto = None
+    _supported_sub_protocols = [ETHProtocol]
+
+    @property
+    def eth_proto(self):
+        assert len(self.enabled_sub_protocols) > 0
+        if self._eth_proto is None:
+            for proto in self.enabled_sub_protocols:
+                if proto.name == ETHProtocol.name:
+                    self._eth_proto = proto
+            if self._eth_proto is None:
+                raise Exception(
```

Comment:
Anything better than a bare *Exception* that can be raised here?

Reply:
I'm not changing it here as this whole thing is a temporary hack anyway.